lib/host.js

```diff
@@ -1,20 +1,30 @@
 var _ = require('lodash-contrib');
 var request = require('request');
+var futils = require('./fhutils');
 
-module.exports = function host(cb) {
-  var url = 'https://' + process.env.FH_MILLICORE + '/box/srv/1.1/ide/apps/app/hosts';
-  var data = {
-    "guid": process.env.FH_INSTANCE,
-    "env": process.env.FH_ENV
+module.exports = function(cfg) {
+  if (!cfg) {
+    return;
   }
 
-  request.post({
-    url: url,
-    json: true,
-    body: data
-  }, function (err, res, body) {
-    if (err) return cb(err);
+  var fhutils = new futils(cfg);
+  var millicoreProps = fhutils.getMillicoreProps();
 
-    return cb(err, _.getPath(body, "hosts.url"));
-  });
+  return function host(cb) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)